README.md

```diff
@@ -42,15 +42,22 @@ specified within the endpoint's link_id parameter.
 **/deactivate/<link_id>**
 Deactivates the active link pointed to via the id specified
 within the endpoint's link_id parameter.
+If deactivation date had been set, it will not be carried over.
 
 **/activate/<link_id>**
 Activates the inactive link pointed to via the id specified
 within the endpoint's link_id parameter.
+If activation date has been set, it will not be carried over.
 
 **/delete/<status>/<link_id>**
 Status = {'active', 'inactive'}.
-Deletes the link with the status and the id specified by the
-endpoint's respective parameters status and link_id.
+Deletes the link with the status and the id specified by
+the endpoint's respective parameters status and link_id.
+
+**/reset/<status>/<link_id>**
```

Comment:
Hmmm, I see the point. With the current logic of the forms, it isn't possible to clear the password so we need this endpoint. But I wonder whether this intuitive enough for the user. What do you think about about adding a checkbox "password protected" to the forms instead of this endpoint?

I mean, the form could look like this:

```jinja
      <ul>
        <li><input name=endpoint type=text></li>
        <li><input name=url type=text></li>
        <li><input name=password_protected type=checkbox></li> <!-- !!!!!!! -->
        <li><input name=password type=password></li>
        <li><input name=switch_date type=text></li>
        <li><input type=submit></li>
      </ul>
```

Than the business logic could look like this:

```python
if form.password_protected.data:  # or however WTF forms represent checkboxes...
    password = form.password.data
else:
    password = None
...
await conn.execute(
    table.update()
        .where(table.columns['id'] == link_id)
        .values(url=..., switch_date=..., password=password))
```

Just an idea... We can change this any time later...

Reply:
The password reset button is only visible for links that have a password. The message above the password field varies as well.
I'd leave it as is for now and go back if it turns out unintuitive.